nikola/utils.py

```diff
@@ -1255,6 +1255,20 @@ class ExtendedRSS2(rss.RSS2):
 
     xsl_stylesheet_href = None
 
+    def __init__(self, itunes_author=None, itunes_summary=None,
+                 itunes_name=None, itunes_email=None,
+                 itunes_image=None, itunes_categories=None,
+                 itunes_explicit=None, **kwargs):
+        self.itunes_author = itunes_author
+        self.itunes_summary = itunes_summary
+        self.itunes_name = itunes_name
+        self.itunes_email = itunes_email
+        self.itunes_image = itunes_image
+        self.itunes_categories = itunes_categories
+        self.itunes_explicit = itunes_explicit
+
+        return rss.RSS2.__init__(self, **kwargs)
```

Comment:
Don’t use `return` here, instead call `rss.RSS2.__init__` at the start of this function. (old-style class in Python 2, so `super()` does not work(